src/components/addBook.js

```diff
@@ -15,14 +15,14 @@ const AddBook = () => {
   };
 
   return (
-    <div>
+    <div className="add-book-cont">
       <form onSubmit={handleSubmit}>
         <h2>Add a Book</h2>
-        <p>Title:</p>
-        <input type="text" id="title" placeholder="Title" required />
-        <p>Author:</p>
-        <input type="text" id="author" placeholder="Author" required />
-        <button type="submit">Add New Book</button>
+        <div className="input-cont">
+          <input type="text" id="title" placeholder="Book Title" required />
+          <input type="text" id="author" placeholder="Author" required />
+          <button type="submit">Add New Book</button>
+        </div>
```

Comment:
- [ ] KIndly add the input for category according to the Zeplin's design. You don't have to implement the function, just add it as a placeholder. 👍

Reply:
Just added a placeholder for the Category, thank you!